static/docs/commands-reference/import-url.md

```diff
@@ -252,7 +252,7 @@ Let's now manually reproduce
 _Get Started_ project. Download the sample source code archive and unzip it:
 
 ```dvc
-$ wget https://dvc.org/s3/get-started/code.zip
+$ wget https://dvc.org/s3/code/get-started/code.zip
```

Comment:
we can do `code.dvc.org/get-started` and `data.dvc.org/get-started` everywhere. We'll need to update the server.js accordingly/

Reply:
Yup, I also realized that after the fact :P  But that's what 30f3db2 is for. I just hope no other repo or project is using dvc.org/s3 redirects because I completely removed that route.

Reply:
Announced in https://iterativeai.slack.com/archives/CB41NAL8H/p1566015493031100